.github/merge_rules.yaml

```diff
@@ -86,6 +86,17 @@
   - pull
   - inductor
 
+- name: OSS CI / pytorchbot / slow tests
+  patterns:
+  - test/slow_tests.json
+  approved_by:
+  - pytorchbot
+  ignore_flaky_failures: false
+  mandatory_checks_name:
+  - EasyCLA
+  - Lint
+  - pull
```

Comment:
Also `- slow`?

Reply:
Any idea why trunk  isn't part of most of these rules?

Reply:
Not sure

Reply:
I think trunk used to mean only jobs running in trunk, not PR.  Then we make it running as part of merge process, so it's not here.